addons/calendar/models/calendar_event.py

```diff
@@ -1296,3 +1296,24 @@ def _get_public_fields(self):
             'id', 'active', 'allday',
             'duration', 'user_id', 'interval', 'partner_id',
             'count', 'rrule', 'recurrence_id', 'show_as', 'privacy'}
+
+    def _notify_get_recipients_groups(self, message, model_description, msg_vals=None):
```

Comment:
Why are we overriding the "_notify_get_recipients_groups" on the calendar.event when we want to add buttons to templates targeting the calendar.event.attendee model? How does that work?

Reply:
As the 'mail.thread' is inherited here so that's why I had done that part here